src/components/RecipeGallery.js

```diff
@@ -0,0 +1,74 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+
+function RecipeGallery({userInput}) {
+
+    const [recipeResults , setRecipeResults ] = useState([]);
+
+    const [recipeSearch, setRecipeSearch] = useState(null);
+
+    if(recipeSearch !== null) {
+        setRecipeSearch(userInput)
+    }
+
+    useEffect(() => {
+        axios({
+            url:'https://api.edamam.com/api/recipes/v2',
+            method: 'GET',
+            dataResponse: 'json',
+            params: {
+                app_key: '4e4719780880769db9a5c0da2940752b',
+                app_id: '78009ee3',
+                q: userInput,
+                foodId: '',
+                type: 'public'
+            },
+        }).then((results) => {
+            console.log(results.data)
+            setRecipeResults(results.data.hits);
+        })
+    }, [userInput])
+
+
+
+    return (
+        
+        <section>
+    
+             <ul className="recipeCards wrapper">
+                {
+                    recipeResults.map( (recipeObject, index) => {
```

Comment:
Was there an id or some other unique property that was part of each recipe object in the api that could be used for the li key as an alternative to index?